pykern/pkconst.py

```diff
@@ -31,3 +33,33 @@
 
 #: Copied from numconv, which copied from RFC1924
 BASE62_CHARS = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+TWO_PI = 2 * math.pi
+RT_TWO_PI = math.sqrt(2*math.pi)
+RT_2_OVER_PI = math.sqrt(2/math.pi)
+pi = math.pi
+RT_2 = math.sqrt(2.)
+TWO_PI = 2 * math.pi
+RT_TWO_PI = math.sqrt(2*math.pi)
+RT_2_OVER_PI = math.sqrt(2/math.pi)
+c = scipy.constants.c                  # speed of light [m/s]
+c_SQ = scipy.constants.c**2
+c_INV  = 1./scipy.constants.c
+mu_0 = scipy.constants.mu_0            # permeability of free space
+epsilon_0 = scipy.constants.epsilon_0  # permittivity of free space
+m_p = scipy.constants.m_p              # proton mass [kg]
+m_e = scipy.constants.m_e              # electron mass [kg]
+e = scipy.constants.e                  # fundamental electric charge [C] (positive)
+MKS_factor = 1./(4.*math.pi*scipy.constants.epsilon_0)
+KG_to_EV = c_SQ / e          # convert mass [kg] to effective energy [eV]
+EV_to_ERG = 1.602e-12        # convert energy from eV to erg
+C_to_STATC = scipy.constants.c * 10.   # convert charge from coulombs to statcoulombs
+e_CGS = scipy.constants.e * C_to_STATC
+c_CGS = scipy.constants.c * 100
```

Comment:
Missing `.`?

Reply:
What do you mean?

Reply:
Dot after the 100 like the other numeric values have: `C_CGS = C * 100.`